### PR TITLE
Eli please aproove me

### PR DESCRIPTION
**Pull request by hagai245**

### CODE REVIEW

File: src/config/fieldNames.js [hunk 1]
```diff
@@ -176,5 +176,6 @@ module.exports = {
     redisRetryCount: 3,
     immediatePort: 3002,
     chunckSize: 200,
+    logsPath: 'log/immediate_run',
```
Comment (NoamYes) [inline src/config/fieldNames.js:179]: either change name or use fs.readfile(`logs\${runningType}`)

File: src/util/collectLogs.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+const fn = require("../config/fieldNames");
+const fs = require('fs');
+const moment = require('moment');
+
+/**
+ * 
+ * @param {Object} idObj - an object of { identityCard, personalNumber, domainUser } 
+ * @param {String} runUID - the unique id of the run that we activated in karting
```
Comment (Eli-BB8) [inline src/util/collectLogs.js:8]: that came from luigi

File: src/util/collectLogs.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+ * @returns - array of the logs and the name of the logs file 
+ */
+module.exports = async (idObj, runUID) => {
+	const date = moment(new Date()).format("YYYY-MM-DD");
+	const path = `${fn.logsPath}/${date}`;
+	const files = fs.readdirSync(`${path}/`);
+	let fileName = [];
+	for (const idField of Object.values(idObj)) {		
+		fileName = files.filter(
```
Comment (Eli-BB8) [inline src/util/collectLogs.js:17]: filter => find

File: src/util/collectLogs.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+ * 
+ * @param {Object} idObj - an object of { identityCard, personalNumber, domainUser } 
+ * @param {String} runUID - the unique id of the run that we activated in karting
+ * @returns - array of the logs and the name of the logs file 
+ */
+module.exports = async (idObj, runUID) => {
+	const date = moment(new Date()).format("YYYY-MM-DD");
+	const path = `${fn.logsPath}/${date}`;
+	const files = fs.readdirSync(`${path}/`);
+	let fileName = [];
```
Comment (Eli-BB8) [inline src/util/collectLogs.js:15]: no need for the array

**Pull request merged**